Amazigh patron deity

Bannana-Puncakes opened the pull request:
Duplicated the equivalent Bori event etc... and changed for Amazigh and Amazigh reformed. Also made a small change to the Bori localisations for changing your mind once you selected the spirit group in the final event, changed the text from "(Yan Riga) is not for me" to "(Yan Riga) are not for me" as the groups should be plural.

Commit:
Amazigh patron deities

Should be finished. scripting done and event pictures done. Small change to bori localisations, changed is to are because the bori options are groups of spirits and therefore plural.

Commit:
Self review changes

couple of small mistakes fixed with localisations for events assigned to the right place.

Commit:
Merge branch 'master' into Amazigh-patron-deity

faiuwle commented on AfricanReligions/events/afr_misc_events.txt:
```diff
@@ -489,4 +489,230 @@ character_event = {
     }
+}
+
+# Amazigh Selecting Patron Deities events
```
You should put these events in their own file.  I'm moving the Bori ones to their own file in my PR.

Bannana-Puncakes replied:
is afr_amazigh_events.txt ok? and are there any other amazigh events I should move across

faiuwle replied:
Probably should be afr_amazigh_patron_deity_events.txt or something like that - there are other Amazigh events, like the Awussu carnival and the Carthage events, which shouldn't all be lumped into one file.

faiuwle commented on AfricanReligions/events/afr_misc_events.txt:
```diff
@@ -489,4 +489,230 @@ character_event = {
+    option = {
+        name = EVTOPTA_AFRREL061
+
+        hidden_tooltip = { character_event = { id = AFRREL.063 } }
```
After discussion in Discord, we decided that we actually want to do it so that you get the modifier here, and then don't get an option to return from the second event.

Bannana-Puncakes replied:
Are you doing this for the Bori events as well? should I change them as well?

Bannana-Puncakes replied:
Seen in the other PR that you've already done this. Thanks

faiuwle commented on AfricanReligions/localisation/afr_event_texts.csv:
```diff
@@ -2,8 +2,8 @@ EVTOPTEHF23004;The myths of the Dark Continent intrigue me.;;;;;;;;;;;;;x
 EVTOPTA_AFRREL003;Glory to our mother, [Root.RelHead.GetBestName]!;;;;;;;;;;;;;x
-EVTDESC_AFRREL004;To all my faithful,\nFor long the unbelievers have lived in our lands, shared our tables and enjoyed the protection of our spears. This is an example of the infinite tolerance of our faith. However it�s time for these fools who reject the truth of the Jan Gari to repay our generosity. For this reason I authorize the lords to tax their unfaithful peasantry.\nLovingly, your Inna;; ...
-EVTOPTA_AFRREL004;The Inna's word bring joy to my soul� and my pocket.;;;;;;;;;;;;;x
+EVTDESC_AFRREL004;To all my faithful,\nFor long the unbelievers have lived in our lands, shared our tables and enjoyed the protection of our spears. This is an example of the infinite tolerance of our faith. However it’s time for these fools who reject the truth of the Jan Gari to repay our generosity. For this reason I authorize the lords to tax their unfaithful peasantry.\nLovingly, your Inna;; ...
```
You've messed up the encoding again.  You need to figure out how to make your editor only save files in Windows-1252.

faiuwle commented on AfricanReligions/localisation/afr_event_texts.csv:
```diff
@@ -68,45 +68,70 @@ EVTOPTA_AFRREL030;A day that will forever be remembered.;;;;;;;;;;;;;x
+EVTOPTA_AFRREL066;The sun that illuminates all land will let me understand it better.;;;;;;;;;;;;;x
+EVTOPTB_AFRREL066;Magec is not for me.;;;;;;;;;;;;;x
+EVTDESC_AFRREL067;Set is a god of chaos, the desert, the wild storms and of violence among men. In the ancient times when the pharaohs still ruled their river kingdom, they also though he was the god of foreigners, because he loved us Amazighs. Set is a trickster but also a protector who fights a constant battle against death itself.;;;;;;;;;;;;;x
+EVTOPTA_AFRREL067;Give me some of your cleverness lord of chaos.;;;;;;;;;;;;;x
```
Needs a comma after "cleverness".

faiuwle commented on AfricanReligions/localisation/afr_general_localizations.csv:
```diff
@@ -213,6 +213,8 @@ vanilla_pick_main_deity;Pick Patron Deity;;;;;;;;;;;;x
 vanilla_pick_main_deity_desc;There are many gods and goddesses who rule over different aspects of our life. While all of them should be honored properly, it is common for people to pick a deity whom they pray and sacrifice to more often.;;;;;;;;;;;;x
 bori_pick_main_deity;Pick Patron Bori;;;;;;;;;;;;x
 bori_pick_main_deity_desc;It is good to have a protective spirit, therefore it would be a good idea to choose one.;;;;;;;;;;;;x
+amazigh_pick_main_deity;Pick Patron Deity;;;;;;;;;;;;x
```
I don't think we need different localization for this one, so we don't need these localization keys.  Amazigh will just use the regular "not Bori" localization.

Bannana-Puncakes replied:
There's a different decision localisation on the spreadsheet for the decision but I can remove it if you wish.

faiuwle replied:
I think it's more important to be consistent here.  I mainly wanted different text for Bori because the bori are not really deities, per se.

faiuwle commented on AfricanReligions/localisation/afr_event_texts.csv:
```diff
@@ -68,45 +68,70 @@ EVTOPTA_AFRREL030;A day that will forever be remembered.;;;;;;;;;;;;;x
+EVTDESC_AFRREL063;Tanit is the mother goddess who brings the rains to the arid lands and gives fertility to women without children. Tanit was first worshiped among both Numidians and Phoenicians, eventually becoming the patroness of Carthage. From there, her cult spread to all the Mediterranean.;;;;;;;;;;;;;x
+EVTOPTA_AFRREL063;Her blessing will bring happiness and love to my family.;;;;;;;;;;;;;x
+EVTOPTB_AFRREL063;Tanit is not for me.;;;;;;;;;;;;;x
+EVTDESC_AFRREL064;Änti was a powerful giant, born of the very earth, who became the most formidable fighter in the whole world because while he was in contact with his mother he became invincible. The Greek men tell that the giant was defeated by Heracles, but we the Amazigh know that this story is nothing but foolishness. Änti simply returned to his mother's womb, in the Tangier, where he patien ...
```
This text is too long.

faiuwle commented on AfricanReligions/localisation/afr_event_texts.csv:
```diff
@@ -68,45 +68,70 @@ EVTOPTA_AFRREL030;A day that will forever be remembered.;;;;;;;;;;;;;x
+EVTOPTA_AFRREL064;With the blessing of Änti I will also be invincible.;;;;;;;;;;;;;x
+EVTOPTB_AFRREL064;Änti is not for me.;;;;;;;;;;;;;x
+EVTDESC_AFRREL065;Gurzil is the great god-bull of war. Many have been the warriors who have sought his blessing on the battlefield. Ierna dedicated his life to the faith so that the bull would protect him against the Greek emperor, and Dihya bore his image when she confronted the Islamic invaders.;;;;;;;;;;;;;x
+EVTOPTA_AFRREL065;I raise a prayer to the great bull, guide my armies in war and bring death to my enemies.;;;;;;;;;;;;;x
```
This text is too long.

faiuwle commented on AfricanReligions/events/afr_misc_events.txt:
```diff
@@ -489,4 +489,230 @@ character_event = {
+    option = {
+        name = EVTOPTC_AFRREL062
+
+        hidden_tooltip = { character_event = { id = AFRREL.060 } }
```
This goes to the wrong event.  It should go to 61.

Bannana-Puncakes replied:
I had it go back to the opening event of the chain so you could then not choose any patron deity if you wanted to and exit the event chain but if this is incorrect I'll change it

faiuwle replied:
I think that'd be a rare case.  The vanilla events don't even have a choice to exit the event chain, I don't think.

faiuwle reviewed (changes requested):
Looks good, just a couple of things to clean up.

Commit:
PR changes 1

Moved amazigh event to it's own file. Reformatted event chain so you get the modifier from the second event. Should have fixed encoding. One grammer cahnge for the set event localisation. Rest of changes to be done once I've confirmed some things.

Commit:
PR changes 2

That should be the encoding fixed.

Commit:
PR changes 3

Final set of PR changes

faiuwle committed:
Fixed Änti event text

faiuwle committed:
Merge branch 'master' into Amazigh-patron-deity
